apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -34,7 +34,11 @@ const FreeResponseResponses = ({responses, showStudentNames}) => {
     pinResponse = undefined,
     unpinResponse = undefined
   ) => (
-    <div key={response.user_id} className={styles.studentResponseBlock}>
+    <div
+      key={response.user_id}
+      className={styles.studentResponseBlock}
+      id="student-response"
```

Comment:
Checking my own understanding here... won't this ID be multiple places in the DOM when there are multiple responses?  My main understanding is that screen readers rely on the ids to be unique, but perhaps that is mitigated here in some other way?

Reply:
Good catch, this was just an oversight.